app/components/containers/containers.html

```diff
@@ -112,7 +112,7 @@
               <td ng-if="state.displayIP">{{ container.IP ? container.IP : '-' }}</td>
               <td ng-if="applicationState.endpoint.mode.provider === 'DOCKER_SWARM'">{{ container.hostIP }}</td>
               <td>
-                <a ng-if="container.Ports.length > 0" ng-repeat="p in container.Ports" class="image-tag" ng-href="http://{{p.host}}:{{p.public}}" target="_blank">
+                <a ng-if="container.Ports.length > 0" ng-repeat="p in container.Ports" class="image-tag" ng-href="http://{{ URLPublish && URLPublish !== '_' ? URLPublish : p.host }}:{{p.public}}" target="_blank">
```

Comment:
I do not like this, it should be a check to determine if `URLPublish` is defined otherwise use `p.host`.